irispy/iris_tools.py

```diff
@@ -21,6 +21,9 @@
 from sunpy.util.net import check_download_file
 from ndcube import NDCube
 
+from astropy.time import Time
```

Comment:
Move this line among the other astropy imports above.

Reply:
Yup, moved `from astropy.time import Time` to above along with the other astropy imports.